app/main/controllers/template/rtmedia-actions.php

```diff
@@ -359,7 +369,7 @@ function rtmedia_link_in_footer() {
 
 	if ( $link ) {
 
-		$aff_id = ( '' != $option['rtmedia_affiliate_id'] ) ? '&ref=' . $option['rtmedia_affiliate_id'] : '';
+		$aff_id = ( '' !== $option['rtmedia_affiliate_id'] ) ? '&ref=' . $option['rtmedia_affiliate_id'] : '';
```

Comment:
The option was removed in https://github.com/rtMediaWP/rtMedia/commit/c53c1fa5a4e741d38840d1dfe4fa17f9910a547d, we can remove this now.

Reply:
okay